introduction/prototyping_gitbook.md

```diff
@@ -0,0 +1,17 @@
+# Prototyping gitbook
```

Comment:
Well, that's supposed to be the same document in fact, I just made a little mistake when I wrote the link. I've just corrected it.